File: graphql/src/test/java/ru/romanow/protocols/graphql/GraphQLApplicationTest.java
package ru.romanow.protocols.graphql;

import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.context.annotation.Import;
import ru.romanow.protocols.graphql.config.DatabaseTestConfiguration;

@SpringBootTest
@Import(DatabaseTestConfiguration.class)
class GraphQLApplicationTest {

    @Test
    void testApp() {

    }
}

File: graphql/src/test/java/ru/romanow/protocols/graphql/GraphQLApplicationTest.kt
package ru.romanow.protocols.graphql

import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.context.annotation.Import
import org.springframework.test.context.ActiveProfiles
import ru.romanow.protocols.graphql.config.DatabaseTestConfiguration

@ActiveProfiles("test")
@SpringBootTest
@Import(DatabaseTestConfiguration::class)
internal class GraphQLApplicationTest {

    @Test
    fun testApp() {
    }
}
